### PR TITLE
Update audio-conferencing-common-questions.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6877
Updated dial-pad commands

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/audio-conferencing-common-questions.md [hunk 1]
```diff
@@ -127,6 +127,8 @@ We are working on a solution that will let the user access and reset a PIN witho
 - *6 (Mute/unmute themselves)
     
 - *1 (Plays the descriptions of dial-pad commands that are available) 
+
+- *2, *21, *22 (Lobby and Entry notifications)
```
Comment (JohanFreelancer9) [inline Teams/audio-conferencing-common-questions.md:131]:
```suggestion
- *2, *21, *22 (Lobby and entry notifications)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**